lib/engine/step/base.rb

```diff
@@ -16,6 +17,14 @@ def initialize(game, round, **opts)
         @round = round
         @opts = opts
         @acted = false
+        @name = name
+      end
+
+      def name
```

Comment:
why do you need this? can't you just check that it's a SpecialTrack?

Reply:
On my branch with the more extensive ability timing changes, it is assumed that when an ability's `when` refers to `track`, that really means the `Track` step, so I added `#name` here to keep the ability `when` consistent. Switched to using `.is_a?(SpecialTrack)` in 54d5d9e6, and will give some more thought about how to handle this on my branch (which has a few other things that need cleaning up before I open a PR for it anyway)